src/tools/rados/rados.cc

```diff
@@ -1428,7 +1435,10 @@ static int do_get_inconsistent_cmd(const std::vector<const char*> &nargs,
     }
     items.clear();
   }
-  formatter.flush(cout);
+  if (opened) {
+    formatter.close_section();
```

Comment:
This close_section() corresponds to the new "formatter.open_object_section("info");" at line 1420.